### ISSUE TITLE
[Toast] Remaining tasks

### ISSUE THREAD

**Issue by jjenzz** [radix-ui/primitives#1171]
- [ ] 🐛 Toast viewport will be `aria-hidden` when `modal={true}` comps open
- [ ] 🐛 Toast hotkey doesn’t work from `modal={true}` comps due to trapped focus
- [ ] ✨ When keyboard/SR users enter toast viewport, they can't get back to their previously focused element easily

## Toast viewport will be `aria-hidden` when `modal={true}` comps open

We have agreed this is a non-issue for a11y currently because the `aria-live` version of the toast is appended to the body (not viewport) so any toasts that are opened from a modal will still be announced. 

Given than toasts should not be designed to acquire a response, there shouldn't be a **need** to access the viewport. It's more of a "nice to have".

## Toast hotkey doesn’t work from `modal={true}` comps due to trapped focus

Is this a bug or expected? Still need to discuss here.

## When keyboard/SR users enter toast viewport, they can't get back to their previously focused element easily

SR users should be able to use their SR tools to navigate to most tabbable things allowing them to move back to where they were relatively quickly. For keyboard users though, they will be forced to tab through the entire tree again to get back to where they were.

It'd be good to figure out some improvements here.
